src/executors/java_localstack_postgres.yml

```diff
@@ -35,6 +37,7 @@ parameters:
     default: "sqs,sns"
     type: string
   postgres_tag:
+    default: "16.1"
```

Comment:
Setting these so this executor can be used as the default in database_schema_report.

It seems impossible to specify an executor which requires parameters as a default